script/robot-presubmit

```diff
@@ -79,6 +79,9 @@ done
 
   # Tests
 
+  bazel test --test_output=errors //start_esp/test/... \
```

Comment:
I think this script is used.

https://github.com/cloudendpoints/esp/blob/master/script/presubmits

You can remove this script.  It is not used any more

Reply:
The PR has been changed to invoke the start_esp tests from the presubmits script.